src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -711,14 +743,307 @@ class C
     static void Main()
     {
         int i = checked(default);
-        int j = checked(default + 4);
-        System.Console.Write($""{i} {j}"");
+        System.Console.Write($""{i}"");
     }
 }
 ";
             var comp = CreateStandardCompilation(source, parseOptions: TestOptions.Regular7_1, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
-            CompileAndVerify(comp, expectedOutput: "0 4");
+            CompileAndVerify(comp, expectedOutput: "0");
+        }
+
+        [Fact]
+        public void InChecked2()
+        {
+            string source = @"
+class C
+{
+    static void Main()
+    {
+        int j = checked(default + 4);
```

Comment:
>default + 4 [](start = 24, length = 11)

It would be good to test what does SemanticModel return as an operator symbol for this scenario.